crates/ide/src/inlay_hints.rs

```diff
@@ -52,6 +52,42 @@ pub struct InlayHintsConfig {
     pub closure_style: ClosureStyle,
     pub max_length: Option<usize>,
     pub closing_brace_hints_min_lines: Option<usize>,
+    pub fields_to_resolve: InlayFieldsToResolve,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct InlayFieldsToResolve {
+    pub client_capability_fields: Vec<String>,
```

Comment:
Can we inline the different resolve fields as `bool`s instead of keeping them as a vec?

Reply:
Nice one, that allows to return `..DISABLED_CONFIG` bit back and reduce the diff, thank you.